### PR TITLE
fix: kernel logger guide md typos

### PR DESCRIPTION
**Pull request by MarkoMin**
Those were rendered as text, not as a link to a type.

### CODE REVIEW

File: lib/kernel/doc/guides/logger_chapter.md [hunk 1]
```diff
@@ -428,8 +428,8 @@ The primary Logger configuration is a map with the following keys:
 
 - **`filters = [{FilterId,Filter}]`** - Specifies the primary filters.
 
-  - `FilterId = ``t:logger:filter_id/0`
-  - `Filter = ``t:logger:filter/0`
+  - `FilterId = `t:logger:filter_id/0`
+  - `Filter = `t:logger:filter/0`
```
Comment (garazdawi) [inline lib/kernel/doc/guides/logger_chapter.md:432]:
```suggestion
  - `FilterId = ` `t:logger:filter_id/0`
  - `Filter = ` `t:logger:filter/0`
```

File: lib/kernel/doc/guides/logger_chapter.md [hunk 2]
```diff
@@ -493,8 +493,8 @@ The configuration for a handler is a map with the following keys:
 
 - **`filters = [{FilterId,Filter}]`** - Specifies the handler filters.
 
-  - `FilterId = ``t:logger:filter_id/0`
-  - `Filter = ``t:logger:filter/0`
+  - `FilterId = `t:logger:filter_id/0`
+  - `Filter = `t:logger:filter/0`
```
Comment (garazdawi) [inline lib/kernel/doc/guides/logger_chapter.md:497]:
```suggestion
  - `FilterId = ` `t:logger:filter_id/0`
  - `Filter = ` `t:logger:filter/0`
```

**Comment by MarkoMin** [comment]
I just grepped ``t: over entire codebase and it seems that there are more issues causing links to be missing.

Do you want to me to include fixes in this PR or in separate?

**Comment by garazdawi** [comment]
Good idea! Feel free to include it in this PR. Also, please squash into a single commit.

If you could base the fixes on `maint` that would be great so that we get them out into 27.2 and don't have to wait for 28.

**Comment by MarkoMin** [comment]
Squashed commits.

Not really sure if this is the right way to change base. If not, should I just open a new PR targeting maint?

**Comment by garazdawi** [comment]
Do `git fetch https://github.com/erlang/otp maint && git rebase --onto FETCH_HEAD 0c8c61a` and then forcepush that and you should get the right thing.

**Comment by garazdawi** [comment]
Or just cherry-pick the commit onto maint and force push it to your branch.

File: lib/kernel/doc/guides/logger_chapter.md [hunk 3]
```diff
@@ -470,15 +470,15 @@ Logger API functions that apply to handler configuration are:
...
+- **`id =` `t:logger_handler:id/0`** - Automatically inserted by Logger. The
   value is the same as the `HandlerId` specified when adding the handler, and it
   cannot be changed.
 
 - **`module = module()`** - Automatically inserted by Logger. The value is the
   same as the `Module` specified when adding the handler, and it cannot be
   changed.
 
-- **`level = ``t:logger:level/0`` | all | none`** - Specifies the log level for
+- **`level =` `t:logger:level/0`` | all | none`** - Specifies the log level for
```
Comment (garazdawi) [inline lib/kernel/doc/guides/logger_chapter.md:481]:
```suggestion
- **`level =` `t:logger:level/0` ` | all | none`** - Specifies the log level for
```

**Review by garazdawi — COMMENTED** [review]
You missed one, but otherwise it looks good.

**Comment by garazdawi** [comment]
thanks!

**Pull request merged**